business/workloads.go

```diff
@@ -318,6 +328,41 @@ func (in *WorkloadService) getParsedLogs(namespace, name string, opts *LogOption
 			entry.Severity = strings.ToUpper(severity)
 		}
 
+		// If this is an istio access log, then parse it out. Prefer the access log time over the k8s time
+		// as it is the actual time as opposed to the k8s store time.
+		if opts.IsProxy {
+			engardeParser := parser.New(parser.IstioProxyAccessLogsPattern)
```

Comment:
I've noticed that the new parser is more heavy and it's including a new lag.
Is this new parser something that perhaps can be added as a global var ?

Reply:
@lucasponce I suppose we could make it optional.  Do you have any numbers on the measured lag?  I did some informal tests and it didn't seem too bad.  But of course it will increase with the number of access log entries.  I went back and forth on whether to pre-parse all of the access logs, because I knew it would add fetch time and the parsed data may not be viewed.  But I really wanted the parsed timestamp (for the reasons I explained in my UI comment), and so I decided to take the hit.   An alternative would be to just parse an entry on-demand, and introduce a new API service to do it.  But I'd still probably try and pre-parse the timestamp as the integrated view is not as useful using strict k8s timestamps.